source/userguide/kb.rst

```diff
@@ -716,6 +716,38 @@ In the example below, we search for restaurants whose names best match ``firetra
    We can set the ``size`` parameter of the :meth:`execute()` method to specify the maximum number of records.
 
 
+.. _question_answerer_config:
+
+QuestionAnswerer Configurations
+-------------------------------
+
+To override MindMeld's default QuestionAnswerer configuration with custom settings, you can edit the app configuration file by adding a dictionary of custom setting named :data:`QUESTION_ANSWERER_CONFIG`. If no such dictionary is present in the ``config.py`` file, MindMeld loads a QuestionAnswerer with default settings. The following are the default settings:
+
+.. code-block:: python
+
+  QUESTION_ANSWERER_CONFIG = {
+      'model_type': 'keyword'
+  }
+
+The ``model_type`` can be one of 'keyword' or 'text'. While specifying the former optimizes your QuestionAnswerer search for keywords or short spans of text, the latter optimizes for searching on larger paragraphs or passages of unstructured text. In order to leverage embeddings based semantic matching along with surface-level text features, you can specify one of three model types- 'embedder', 'embedder_text' and 'embedder_keyword'. For leveraging appropriate embeddings, you can specify embedder configurations in the :data:`QUESTION_ANSWERER_CONFIG` with an additional key 'model_settings'. For full details on using embedders in QuestionAnswerer, check the :ref:`Leveraging semantic embeddings <semantic_embeddings>` section below. Note that the specified ``model_type`` is also the default ``query_type`` for all QuestionAnswerer calls in your application, but you can always pass in a different query type to your ``qa.get()`` command as desired.
+
+.. _non_elasticsearch_question_answerer:
+
+QuestionAnswerer without Elasticsearch backend
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Search operations of QuestionAnswerer are by default backed by Elasticsearch. While it is recommended to keep using Elasticsearch backend for QuestionAnswerer, it might not always be feasible to use it due to some or the other constraints. In such cases, one can easily disable Elasticsearch usage by setting a disable-key ``"use_elastic_search": False`` in the ``QUESTION_ANSWERER_CONFIG`` as follows.
+
+.. code-block:: python
+
+  QUESTION_ANSWERER_CONFIG = {
+      'model_type': 'keyword',
+      'use_elastic_search': False
+  }
+
+You can set this key irrespective of leveraging embedders or not. If unspecified, this key will be set to ``True``. It is noteworthy that the search results obtained with and without Elasticsearch might have minor differences between them.
```

Comment:
Can we also briefly mention what happens in the case of non-elasticsearch question answerer?

Reply:
Sure. Can do that. I have few questions though:    
1) Do you want me to mention that we re-purpose TF-IDF and Embedder Resolvers to do the matching or would that confuse the reader? 
1) I can also mention in which folder are the resources cached for re-loading. That should be good right?
1) In this resolver, users can also pass KBs as list of strings which is currently not allowed for ES based QA. Should we mention that? (ref. [Issue 291](https://github.com/cisco/mindmeld/issues/291)) I can look into how we can entertain list of strings in ES based QA too but that might take bit more time.

@vembar

Reply:
I think it's best to not bring in the resolver since that will only confuse the user. Yeah, just the details about where and how it is stored and a one liner or so about the ranking function should be good.